### PR TITLE
Contact layout

### PR DESCRIPTION
**Pull request by anton25360**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -92,3 +98,30 @@ h1{
 	display:flex;
 	flex-direction:row;
 }
+
+/*CONTACT SECTION*/ /*CONTACT SECTION*/ /*CONTACT SECTION*/ /*CONTACT SECTION*/ /*CONTACT SECTION*/
+
+.wrapperContact{
+	margin: 0 10%;
+	display: flex;
+	flex-flow: column ;
```
Comment (ashleycoles) [inline style.css]: Remove the extra space before the semi colon

File: style.css [hunk 2]
```diff
@@ -92,3 +98,30 @@ h1{
...
+
+.wrapperContact a{
+	display:inline-block;
+	text-decoration:none;
+	font-size:35px;
+	margin:5px;
+}
+
+.wrapperContact h2{
+	display: inline-block;
```
Comment (ashleycoles) [inline style.css]: Change all of the CSS rules to be formatted like this line. 

property: rule;

rather than

property:rule;

File: style.css [hunk 3]
```diff
@@ -92,3 +98,30 @@ h1{
...
+	font-size: 3vw;
+	margin: 2px;
+}
+
+.icon{
+	height:100px !important;
```
Comment (ashleycoles) [inline style.css]: Using !important is evil and should be avoided at all costs, can you achieve the same thing by making the selector more specific?

Comment (anton25360) [reply style.css]: !important rule not needed. Has been removed.

**Pull request merged**